### PR TITLE
Amber Network Entity Submission

### PR DESCRIPTION
**Pull request by ravenac95**
Adds:
* Migrating entity docs

Updates:
* Updates "Joining the Testnet" to be "Running a Node on Amber Network"
* Updates a variety of links

### CODE REVIEW

File: src/operators/current-testnet-parameters.md [hunk 1]
```diff
@@ -179,8 +179,7 @@ Upgrades] guide.
 **`/serverdir/etc/config.yml` Required Changes**
 
 If you've deployed before, we changed the storage backend from `boltdb` to
-`badger`. See the [Joining the Testnet](./joining-the-testnet.md) Docs for the
-update.
+`badger`.
```
Comment (ravenac95) [inline src/operators/current-testnet-parameters.md:182]: I had to remove the dead link

File: src/operators/the-quest-rules.md [hunk 2]
```diff
@@ -72,28 +72,28 @@ you to read through the [proposed incentive structure][incentives-proposal]
 for the Oasis Network.
```
Comment (ravenac95) [inline src/operators/the-quest-rules.md:72]: Not sure what to do about this file. I had to remove the dead link on it.

File: src/operators/migrating-questnet-entities.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+* Your `/localhostdir`
+  * We are assuming that you had previously followed our setup guides and
+    created an entity/node in a directory called `/localhostdir`
+* Your `/localhostdir/entity` containing your `entity.pem` (Your entity private key)
+* oasis-node v20.7
+  * Linux: [oasis-node-linux]
+  * macOS: [oasis-node-macos]
+
+[oasis-node-linux]: https://github.com/oasisprotocol/oasis-core/releases/download/v20.7/oasis_core_20.7_linux_amd64.tar.gz
+[oasis-node-macos]: https://github.com/oasisprotocol/oasis-core/releases/download/v20.7/oasis_core_20.7_macos_amd64.tar.gz
```
Comment (ravenac95) [inline src/operators/migrating-questnet-entities.md:33]: Hrm... should probably add a disclaimer not to run the macos version as a node.

File: src/operators/migrating-questnet-entities.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
+# Migrating QuestNet Entities for the Amber Network
```
Comment (peterjgilbert) [inline src/operators/migrating-questnet-entities.md:1]:
```suggestion
# Migrating Your QuestNet Entity for the Amber Network
```

File: src/operators/migrating-questnet-entities.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+For those who have previously participated in The Quest, we ask that you create
+an entity package for the Amber Network derived from your Quest entity. A
+migration is required due to some breaking changes for the node descriptors used
+between The Quest and the Amber Network.
```
Comment (peterjgilbert) [inline src/operators/migrating-questnet-entities.md:6]:
```suggestion
For those who have previously participated in The Quest, you must create
an entity package for the Amber Network derived from your Quest entity. A
migration is required due to breaking changes in the node descriptor format used
between The Quest and the Amber Network.
```

File: src/operators/migrating-questnet-entities.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+To summarize, the steps required for migration are as follows:
+
+1. Backup all of your entity/node artifacts.
+2. Create a set of new node artifacts.
+    * This will effectively delete your old node. While it might be possible to
+      use the same keys with the Amber Network, for simplicity's sake, we will
```
Comment (peterjgilbert) [inline src/operators/migrating-questnet-entities.md:13]:
```suggestion
      use the same node identity with the Amber Network, for simplicity's sake, we will
```

File: src/operators/migrating-questnet-entities.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+      not include those instructions. If you attempt to do so, it is at your own
+      discretion.
+3. Update your entity with the new node descriptor.
+    * This removes the old node descriptor.
+4. Create and submit your Entity Package
+5. Update your validator with the new node artifacts when the Amber Network
```
Comment (peterjgilbert) [inline src/operators/migrating-questnet-entities.md:19]:
```suggestion
5. Start your node with the new node artifacts when the Amber Network
```

File: src/operators/migrating-questnet-entities.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+    created an entity/node in a directory called `/localhostdir`
+* Your `/localhostdir/entity` containing your `entity.pem` (Your entity private key)
+* oasis-node v20.7
+  * Linux: [oasis-node-linux]
+  * macOS: [oasis-node-macos]
+
+[oasis-node-linux]: https://github.com/oasisprotocol/oasis-core/releases/download/v20.7/oasis_core_20.7_linux_amd64.tar.gz
+[oasis-node-macos]: https://github.com/oasisprotocol/oasis-core/releases/download/v20.7/oasis_core_20.7_macos_amd64.tar.gz
+
+## Step 1: Backup all of your entity artifacts
```
Comment (peterjgilbert) [inline src/operators/migrating-questnet-entities.md:35]:
```suggestion
## Step 1: Backup all of your entity/node artifacts
```

File: src/operators/migrating-questnet-entities.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+
+## Step 3: Update your entity with the new node descriptor
+
+Now that we have initialized a new node, we need to add it to the entity
+descriptor so that it can properly register itself when the node starts up.
+
+Execute the following command in the `/localhostdir/node` directory:
+
+```bash
+oasis-node registry entity update \
```
Comment (peterjgilbert) [inline src/operators/migrating-questnet-entities.md:87]: Do we also need to remove the old node from the entity descriptor?

File: src/operators/migrating-questnet-entities.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+## Step 4: Create and submit your Entity Package
+
+Once you've completed the steps above, follow the guide in [Creating an Entity
+Package] _before the beginning of the Amber Network_.
+
+[Creating an Entity Package]: ./creating-an-entity-package.md
+
+## Step 5: Update Your Validator
```
Comment (peterjgilbert) [inline src/operators/migrating-questnet-entities.md:99]:
```suggestion
## Step 5: Update your node
```

File: src/operators/running-node-on-amber-network.md [hunk 11]
```diff
@@ -93,19 +92,23 @@ This will be needed later when generating transactions.
 
 ### Initializing an Entity
 
-As described in the [Architecture Overview][arch-entity], an entity is
-critical to operating nodes on the network as it controls the stake attached to
-a given individual or organization on the network.
-In the future, we will support using entity keys through HSMs to ensure that
-entity keys cannot be easily compromised.
+::: tip NOTE
+If you previously created an entity for The Quest, we ask that you upgrade that
```
Comment (peterjgilbert) [inline src/operators/running-node-on-amber-network.md:96]:
```suggestion
If you previously created an entity for The Quest, you need to upgrade that
```

File: src/operators/running-node-on-amber-network.md [hunk 12]
```diff
@@ -194,7 +198,9 @@ This will update the entity descriptor in `entity.json` and subsequently the
 
 #### Initializing Additional Nodes
 
-At the time of Public Testnet, the network will only have validators and no
+<!-- TODO... what do with this section -->
```
[the remark this answers is not in the document]
Comment (ravenac95) [reply src/operators/running-node-on-amber-network.md:201]: Ya, probably the best option.

File: src/operators/the-quest-rules.md (hunk 2 above)
Comment (Afauvre) [reply src/operators/the-quest-rules.md:77]: yeah let me update these docs. I think we can just keep the winners section and remove the rest. Will update.

Comment (peterjgilbert) [reply src/operators/the-quest-rules.md:77]: Might not be a bad idea to keep the old content so folks can review if interested.

File: src/operators/migrating-questnet-entities.md [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+Execute the following command in the `/localhostdir/node` directory:
+
+```bash
+oasis-node registry entity update \
+  --signer.dir /localhostdir/entity \
+  --entity.node.descriptor node_genesis.json
+```
+
+You may be wondering if you need to remove the old node from the entity. The
+`oasis-node registry entity update` command only include explicitly specified
```
Comment (peterjgilbert) [inline src/operators/migrating-questnet-entities.md:93]:
```suggestion
`oasis-node registry entity update` command includes only explicitly specified
```

File: src/operators/stake-management.md [hunk 14]
```diff
@@ -248,8 +248,8 @@ Currently, we provide two options for signing transactions:
 
 * Entity's private key stored in a file.
 
-  You will need to create your Entity as described in [Joining the Testnet][
-  create-entity] docs and set the following flags:
+  You will need to create your Entity as described in [Running a Node on the
+  Amber Network][create-entity] docs and set the following flags:
 
   * `--signer file`: Specifies use of the file signer.
```
Comment (ptrus) [inline src/operators/stake-management.md:254]: `--signer file` -> `--signer.backend file` (maybe do a general check through the docs if there are any remaining references to the `--signer` flag)

Comment (ravenac95) [reply src/operators/stake-management.md:254]: 👍

Comment (ravenac95) [inline src/operators/stake-management.md:254]:
```suggestion
  * `--signer.backend file`: Specifies use of the file signer.
```

File: src/operators/migrating-questnet-entities.md (hunk 3 above)
Comment (peterjgilbert) [reply src/operators/migrating-questnet-entities.md:33]: Do you feel like we need to provide this binary? Alternatively, we could help folks through operator slack if they ask.

File: src/operators/the-quest-rules.md (hunk 2 above)
Comment (peterjgilbert) [reply src/operators/the-quest-rules.md:77]: I think @Afauvre's changes have addressed this.

File: src/operators/the-quest-rules.md [hunk 15]
```diff
@@ -1,5 +1,10 @@
 # Rules and Guidelines for The Quest
 
+::: tip NOTE
+The Quest has ended. We are preparing to deploy to the release candidate of the
+Mainnet. See [Amber Network](./amber-network.md) for more details.
+:::
```
[the remark this answers is not in the document]
Comment (peterjgilbert) [reply src/operators/the-quest-rules.md:6]: I think @Afauvre's changes have addressed this.

[the remark this answers is not in the document]
Comment (ravenac95) [reply src/operators/the-quest-rules.md:6]: Let me resolve the conflicts

File: src/operators/migrating-questnet-entities.md (hunk 3 above)
Comment (ravenac95) [reply src/operators/migrating-questnet-entities.md:33]: Ah... good question. Let's go without it.

**Pull request merged**